cinder/volume/drivers/lightos.py

```diff
@@ -420,9 +420,9 @@ def create_cloned_volume(self, volume, src_vref):
             self._create_volume(volume,
                                 src_snapshot_lightos_name=snapshot_name)
         except Exception as e:
-            LOG.warn("Failed to create volume %s from intermediate "
-                     " snapshot %s. Trying to clean up.",
-                     src_volume_name, snapshot_name)
+            LOG.error("Failed to create volume %s from intermediate "
+                      " snapshot %s. Trying to clean up.",
```

Comment:
note, not important but there is white space both after intermediate and before snapshot